types/gulp-javascript-obfuscator/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "javascript-obfuscator": ">=1"
```

Comment:
can this be fixed to some recent version? thx!

Reply:
I changed it to `^3.0.0`, the latest major release